Assets/Scripts/Player/Player.cs

```diff
@@ -83,6 +87,7 @@ public class Player : MonoBehaviour {
 				if (id == playerId && canPerformAction) {
 					canPerformAction = false;
 
+					animator.SetInteger("Action", 2);
```

Comment:
Reset needs to occur here as well, otherwise the animation will just keep repeating.